### DISCUSSION TITLE
"No route to host" in macOS Sequoia 15.0

### DISCUSSION THREAD

**Discussion by tallest1** [transmission-remote-gui/transgui#1477] [General]
I'm getting the error "No route to host" running the GUI in macOS Sequoia 15.0 (connecting to a local server). I can successfully connect to the server using transmission-remote from the command line in macOS. 

I believe it's linked to the new "Local Network" settings under "Privacy & Security" in System Settings. The error is discussed in [dbeaver](https://github.com/dbeaver/dbeaver/issues/35705), [vscode](https://github.com/microsoft/vscode/issues/228862), [flutter](https://github.com/flutter/flutter/issues/150131), and [iacievedit.it](https://dev.iachieved.it/iachievedit/macos-sequoia-no-route-to-host/). 

Their solution involves toggling the switch for the app in the "Local Network" System Preferences pane, but Transmission Remote GUI isn't available there.

**Comment by allindark** [comment]
@tallest1 How did you resolve this problem? I have this one too.

**Reply by 0x3654** [reply]
Try using this version: https://github.com/lighterowl/transgui/releases/tag/v5.18.7.f
It works for me
<img width="500" alt="Screenshot 2024-12-22 at 15 08 50" src="https://github.com/user-attachments/assets/0f74c09b-3e69-4f03-8514-6636b0c991f6" />

**Reply by tallest1** [reply]
Macos eventually prompted me to allow Transmission Remote GUI to find devices on the local network. I'm not sure what made it finally ask. It was then in the list of apps in the Local Network security pane. 

After an update to Macos 15.2, I was right back in the same spot again, though. It was no longer in the Preference Pane and the app is reporting "No Route to Host" again. 

I installed version 5.18.7f per allindark's suggestion and it gave the same error when I opened it the first time. Closing and opening it again led to the prompt from Macos and now it's working again.
